features/adresregels.feature

```diff
@@ -0,0 +1,133 @@
+# language: nl
+
+Functionaliteit: Als gebruiker wil ik samengestelde adresregels
+    zodat ik het samengestelde adres direct en correct kan gebruiken.
+
+    Het adres wordt samengesteld conform NEN 5825:2002. Dit betreft alleen het adresdeel van de adressering, het naamdeel van de adressering komt uit een andere bron (zoals BRP, HR of BRK).
+
+    De adresregels worden samengesteld op basis van de volgende regels
+    - Property adresregel1 wordt samengesteld uit korteNaam + huisnummer + huisletter + huisnummertoevoeging
+    - Property adresregel2 wordt samengesteld uit postcode + woonplaats
+    - Tussen kortenaam en huisnummer wordt een spatie opgenomen
+    - Tussen het huisnummer en de huisnummertoevoeging (wat in de NEN een samenstelling is van huisletter en huisnummertoevoeging, niet gecheiden van elkaar) moet een koppelteken ("-") worden geplaatst indien de huisnummertoevoeging met een cijfer begint, een spatie in alle andere gevallen
+    - Tussen postcode numeriek deel en postcode alfabetisch deel moet een spatie worden geplaatst
+    - Tussen postcode alfabetisch deel en woonplaatsnaam moeten twee spaties worden geplaatst
+
+    Scenario: er is geen huisnummer en geen huisletter
+        Gegeven adres met nummeraanduidingidentificatie "0957200000021241" heeft de volgende gegevens:
+        """
+        "straat": "Annelien Kappeyne van de Coppellostraat",
```

Comment:
Ik denk dat het handiger is om dit in een tabel op te nemen of om het tussen accolades te zetten als hiermee een adres in json wordt bedoeld

Reply:
ik zet er accolades omheen